content/building-a-preview/administer-previews/build-previews.md

```diff
@@ -81,6 +81,21 @@ out multiple Previews. Check out our section on
 [Optimizing Preview builds](../../preview-deep-dive/optimize-preview-builds/) for tips on
 [reducing Preview size](../../preview-deep-dive/optimize-preview-builds/#optimizing-preview-size). {{% /notice %}}
 
+### Pull Request Merging Behavior
+
+When building a preview from a pull request, Tugboat will first merge the branch locally before building the preview. This
+allows you to test your project in the state it will be in after the pull request is merged. In particular, this is useful
+for identifying bugs and regressions that only happen when code in your main or other branch exposes bugs only in combination
+with the changes in the pull request.
+
+Sometimes, it can be useful to test a pull request without first merging code, such as when the only conflicts are in
+generated files like package manager lock files. To trigger a build without a merge:
+
+1. Delete any existing pull request previews.
+2. In Tugboat, build a preview from the _branch_ or _tag_ instead of from a _pull request_. Since there's no implicit merge
+   destination, Tugboat will build without a merge.
+3. In the case of a branch, Tugboat will still post environment URLs to the pull request.
```

Comment:
This may be true of GitHub Status and GitHub Deployments, but I'm not sure if this is true for the Comment, if you have that feature turned on. Also would be helpful to know how this works for Bitbucket and Gitlab. Maybe we just take this point out?

Reply:
Done!